Change global variable username by USERID constant

marlycormar opened the pull request:
Fixes issue #4 
- The global variable `$username` is substituted by `USERID`.
- Tested the behavior in redcabp.

Fixes issue #3
- Set minimum redcap version

Fixes issue #2 
- Add Taryn as an author

Other commits were added per Philip's request (please see comments below). These new commits reflect the change of variable name from `username` to `USERID`.

marlycormar committed:
Change global variable username by USERID constant

marlycormar committed:
Update minimum redcap version number

marlycormar committed:
Add Taryn as an author

pbchase commented:
I think you should also revise https://github.com/ctsit/modify_contact_admin_button/blob/8a69cf6ef683803699609258edcb222fce7fad29/config.json#L63 and the references to "username" in https://github.com/ctsit/modify_contact_admin_button/blob/master/README.md and https://github.com/ctsit/modify_contact_admin_button/blob/master/example_configuration.png

marlycormar committed:
Change variable name username by USERID when reading from the config.json

marlycormar committed:
Change the username variable name in the module configuration to USERID

marlycormar committed:
Update the configuration example to reflect the change of variable name

marlycormar committed:
Change variable name username to USERID and make a comment about the variable content

marlycormar commented:
@pbchase per you request I updated the following documents:
- README.md: changed `username` by `USERID`
- example_configuration.png: updated the configuration example
- config.json: changed `username` by `USERID`
- ExternalModule.php: changed `username` by `USERID` in the list used to set js variables.
